oshc/main/templates/contest_submission.html

```diff
@@ -3,9 +3,9 @@
 {% block content %}
 
 <div class="jumbotron text-xs-center">
-  <div class="container text-center">
-      <h1 class="display-3">Thank You!</h1>
-      <p class="lead"><strong>Your contest has been submitted for review.</strong></p>
+  <div class="container text-center vcenter">
+      <h1 class="display-3 text-center">Thank you!</h1>
```

Comment:
These text-center came back.  :) Only the first one is required.

Reply:
@tapasweni-pathak fixed 👍